metric-schema-markdown/src/main/java/com/palantir/metric/schema/markdown/MarkdownRenderer.java

```diff
@@ -111,6 +110,12 @@ private static void renderLine(
         }
     }
 
+    private static boolean hasComplexTags(List<TagDefinition> tags) {
+        return tags.stream()
+                .anyMatch(definition -> definition.getDocs().isPresent()
+                        || !definition.getValues().isEmpty());
+    }
```

Comment:
Split the code above into a utility function, the difference here is the addition of `definition.getDocs().isPresent() || `